CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.13)
+
+option(FIZZY_TESTING "Enable Fizzy internal tests")
```

Comment:
This is OFF by default, so you should explicitly enable it when developing fizzy, but users can use default options.